cli-toolchain/usage.md

```diff
@@ -346,29 +346,30 @@ Follow the steps below to update to the latest version of the CLI:
 
 The Amplify CLI requires the below IAM policies for performing actions across all categories. You can grant or restrict category permissions by including or removing items from the `Action` section as appropriate. For example, if you wish to restrict operations on the `Auth` category you can remove any of the lines starting with `cognito`.
 
-```terminal
+```json
 {
     "Version": "2012-10-17",
     "Statement": [
         {
             "Sid": "VisualEditor0",
             "Effect": "Allow",
             "Action": [
-                "appsync:*",
-                "apigateway:POST",
+                "amplify:*",
                 "apigateway:DELETE",
                 "apigateway:PATCH",
+                "apigateway:POST",
                 "apigateway:PUT",
+                "appsync:*",
```

Comment:
Do we really need `amplify:*` and `appsync:*`? If we can use the specific policies we should.